app/population/views/dashboard.njk

```diff
@@ -1,10 +1,14 @@
 {% extends "layouts/collection.njk" %}
 
 {% block pageContent %}
-  {% if resultsAsPopulationTable | length  %}
+  {% if resultsAsPopulationTables | length  %}
     <div class="govuk-grid-row">
       <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
-        {{ govukTable(resultsAsPopulationTable) }}
+      {% for table in resultsAsPopulationTables %}
+        {{ govukTable(table) }}
+      {% else %}
+        <h1>No population to display</h1>
```

Comment:
Handling a default, just in case there are no population tables for buggy reasons

Reply:
- `No population to display` -->  locales file
- Should the string be wrapped in an `h1` element? The page will already have a heading defined in the `pageHeader` block.
- (NB. ignore this if this is Andy'd design...) any reason not to reuse the `appMessage` as in `common/templates/collection-as-table.njk`

eg.
![Single_requests_sent_14_to_20_Dec_2020__This_week_](https://user-images.githubusercontent.com/4856/102383899-fc5c1d80-3fc3-11eb-81b9-d3731bd7eb84.png)

Reply:
@solidgoldpig Those are all good suggestions - I'll get them added in :)

No population to display is an edge case the design doesn't cover.

Reply:
Added in appMessage and locales as per your suggestion - thanks :smile: